src/styles/main.scss

```diff
@@ -1,3 +1,6 @@
-body {
-  margin: 0;
-}
+@import "./utils/variables";
+@import "./utils/reset";
+@import "./utils/normalize";
```

Comment:
normalize and resets have to be on the top of the code

Reply:
i remember that @Olexandr6 said that variables should be always on the top